WordPress/Classes/ViewRelated/Portfolios/PortfolioListViewController.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+class PortfolioListViewController: UIViewController {
+    var blog: Blog!
```

Comment:
Please, consider removing the IUO `!` if the **blog** is actually mandatory. Plus, if it cannot be changed, please switch `var` to `let`.